exotic/output_files.py

```diff
@@ -138,15 +138,17 @@ def plate_status(self, plate_status: PlateStatus):
         plate_status_file = self.dir / "temp" / f"PlateStatus_{self.p_dict['pName']}_{self.i_dict['date']}.csv"
         plate_status.writePlateStatus(plate_status_file)
 
-class VSPOutputFiles:
-    def __init__(self, fit, p_dict, i_dict, vsp_params):
+class AIDOutputFiles:
+    def __init__(self, fit, p_dict, i_dict, auid, chart_id, vsp_params):
```

Comment:
Method signature changes ...

Reply:
It's technically not for VSP, but for AID. I can revert back to VSP though if needed.

Reply:
Name change sounds appropriate. Options are to increase major version but that's a bit dramatic. If we version up to 3.5.0 then it indicates itself a semi-major change. So, set version up to 3.5.0.

Reply:
Done in PR #1248